app/assets/javascripts/components/overview/campaign_button.jsx

```diff
@@ -40,26 +44,25 @@ const CampaignButton = createReactClass({
         <button className="button border plus" onClick={this.removeCampaign(campaign.title)}>-</button>
       );
       return (
-        editRows.push(
-          <tr key={`${campaign.id}_campaign`}>
-            <td>{campaign.title}{removeButton}</td>
-          </tr>
-        )
+        <tr key={`${campaign.id}_campaign`}>
+          <td>{campaign.title}{removeButton}</td>
+        </tr>
       );
     });
 
+    const allCampaigns = ['A', 'B', 'C', 'D'];
     let buttonClass = 'button';
     buttonClass += this.props.inline ? ' border plus' : ' dark';
     const buttonText = this.props.inline ? '+' : CourseUtils.i18n('campaigns', this.props.course.string_prefix);
     const button = <button className={buttonClass} onClick={this.props.open}>{buttonText}</button>;
     return (
       <div className="pop__container" onClick={this.stop}>
+        {campaignList}
```

Comment:
I think maybe this makes more sense. I would add the list of current Campaigns + a Remove Button and in the Button underneath, the access to add new campaigns. What do you think of this?
Also, do we need this button or should I create a Select component?
Thanks in advance.

Reply:
I don't completely understand what you're asking. I think keeping the UI the same during this conversation is a good idea.